__tests__/app.test.js

```diff
@@ -58,7 +58,55 @@ describe("/api/articles", () => {
         .expect(200)
         .then((res) => {
           const { articles } = res.body;
-          expect(articles).toBeSortedBy("created_at",{descending: true});
+          expect(articles).toBeSortedBy("created_at", { descending: true });
+        });
+    });
+  });
+
+  describe("GET /api/articles/:article_id/comments", () => {
+    test("should GET 200: responds with array of comments for a specific article ID ", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((res) => {
+          const { comments } = res.body;
+          expect(comments).toBeInstanceOf(Array);
+          expect(comments.length).toBe(11);
+          comments.forEach((comment) => {
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
```

Comment:
I would also assert that the article_id is 1 in this case for every comment

Reply:
The ticket doesnt say to include the article_id in the response, is that incorrect?